apps/embed/src/popup.js

```diff
@@ -0,0 +1,174 @@
+/**
+ * Crowdsignal embed component.
+ */
+class CrowdsignalPopup extends window.HTMLElement {
+	/**
+	 * Wrapper container reference
+	 */
+	#wrapper;
+
+	/**
+	 * Close Button reference
+	 */
+	#closeButton;
+
+	/**
+	 * Reference to the main iframe
+	 */
+	#frame;
+
+	connectedCallback() {
+		const isWpEditor =
```

Comment:
This might not be the best way to check if it's inside the editor, but I didn't find a more elegant way to do that.